pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -22,6 +22,8 @@ in
 rec {
   # Try to keep this generally alphabetized
 
+  bitbucket-server-cli = callPackage ./bitbucket-server-cli { };
```

Comment:
I tried to stick closely to ```gitAndTools.hub```, since I consider this tool analogous in functionality.